.changeset/mighty-shoes-scream.md

```diff
@@ -0,0 +1,39 @@
+---
+'astro': minor
+'@astrojs/cloudflare': patch
+'@astrojs/netlify': patch
+'@astrojs/vercel': patch
+'@astrojs/image': patch
+'@astrojs/deno': patch
+'@astrojs/node': patch
```

Comment:
Marked these as 'patch' because I updated the warning we get when using these adapters with an unexpected output mode.